tests/keras_autolog/test_keras_autolog.py

```diff
@@ -59,7 +60,10 @@ def generator():
 @pytest.mark.parametrize('fit_variant', ['fit', 'fit_generator'])
 def test_keras_autolog_logs_expected_data(keras_random_data_run):
     data = keras_random_data_run.data
-    assert 'acc' in data.metrics
+    if LooseVersion(keras_module.__version__) < LooseVersion('2.3.1'):
```

Comment:
Maybe we don't need this check anymore now that we're only testing Keras 2.3.1? Can simplify the code & probably fix the test that way :)